vizro-core/docs/pages/user-guides/figure.md

```diff
@@ -105,3 +105,238 @@ To add a `Figure` to your page:
 
 If the existing figure functions from [`vizro.figures`](../API-reference/figure-callables.md) do not serve your purpose,
 there is always the option to create a [custom Figure](custom-figures.md).
+
+
+### KPI Cards
+A KPI card is a dynamic card that can display a single value. Optionally, a title, icon and reference value can be added.
+It is a common visual component to display key metrics in a dashboard. Vizro supports two pre-defined KPI card functions:
+
+- [`kpi_card`](../API-reference/figure-callables.md#kpi_card): A KPI card with a single value.
+- [`kpi_card_with_reference`](../API-reference/figure-callables.md#kpi_card_with_reference): A KPI card with a single value and a reference value.
+
+#### Existing KPI Card functions
+!!! example "KPI Card Variations"
+
+    === "app.py"
+        ```py
+        import pandas as pd
+        import vizro.models as vm
+        from vizro import Vizro
+        from vizro.figures import kpi_card, kpi_card_reference
+
+        df = pd.DataFrame([[67434, 65553, "A"], [6434, 6553, "B"], [34, 53, "C"]], columns=["Actual", "Reference", "Category"])
+
+        page = vm.Page(
+            title="KPI Indicators",
+            layout=vm.Layout(grid=[[0, 1, 2, 3], [4, 5, 6, 7], [-1, -1, -1, -1], [-1, -1, -1, -1]]),
+            components=[
+                vm.Figure(figure=kpi_card(data_frame=df, value_column="Actual", title="KPI with value")),
+                vm.Figure(figure=kpi_card(data_frame=df, value_column="Actual", title="KPI with aggregation", agg_func="mean")),
+                vm.Figure(
+                    figure=kpi_card(
+                        data_frame=df,
+                        value_column="Actual",
+                        title="KPI with custom formatting",
+                        value_format="${value:.2f}",
+                    )
+                ),
+                vm.Figure(
+                    figure=kpi_card(
+                        data_frame=df,
+                        value_column="Actual",
+                        title="KPI with icon",
+                        icon="shopping_cart",
+                    )
+                ),
+                vm.Figure(
+                    figure=kpi_card_reference(
+                        data_frame=df,
+                        value_column="Reference",
+                        reference_column="Actual",
```

Comment:
I see why you've done this in order to get the positive and negative styles shown, but I think it's too confusing to swap these columns round.

I would suggest getting the positive/negative to show another way by changing to some different `agg_func` that changes the relationship between reference and actual to be positive vs. negative. This would also provide another example of `agg_func` being different which is nice.

If I've done my sums right then this should work:
```
actual = [1, 3, 5]
reference = [1, 2, 7]
```

sum(actual) < sum(reference)
median(actual) > median(reference)

Or obviously any multiple of those like `[1000, 3000, 5000]` etc.

Reply:
Ah, amazing! Thanks 🙏